src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private final int defaultCircleRadius = 10;
+    private final String defaultCircleColor = "white";
+    private final int figureCount = 5;
```

Comment:
Please use constant naming convention for 'figureCount' by making it uppercase and static final: 'FIGURE_COUNT'.